app/templates/create-masterclass/content/choose-ddat-family.html

```diff
@@ -0,0 +1,34 @@
+{%  extends "base.html" %}
+
+{% block content %}
+<div class="govuk-width-container">
+  <a href="/" class="govuk-back-link">Back</a>
+
+  <main class="govuk-main-wrapper " id="main-content" role="main">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        <form action="/create-masterclass/content/job-family" method="post" novalidate>
+
+          <div class="govuk-form-group">
+            <fieldset class="govuk-fieldset">
+              <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl">
+                <h1 class="govuk-fieldset__heading">
+                  Which DDaT job family is your masterclass related to?
+                </h1>
+              </legend>
+              <span id="DDaT-job-family-link" class="govuk-hint">
+                <a href="https://www.gov.uk/government/collections/digital-data-and-technology-profession-capability-framework#data-job-family">Find out more about the job families</a>
```

Comment:
Should this link be more general, or the text be more specific? 
```suggestion
                <a href="https://www.gov.uk/government/collections/digital-data-and-technology-profession-capability-framework">Find out more about the job families</a>
```

Reply:
I think text should be more specific, so have edited the text to be `Find out more about the DDaT job families`.